readme/Readme.scalatex

```diff
@@ -540,6 +540,45 @@
     @p
       The rest of the code involved in this micro-benchmark can be found in @lnk("PerfTests.scala", "shared/test/scala/scalatags/PerfTests.scala")
 
+   @sect{Utilities}
+
+    @p
+      There's a simple converter to produce simply indented text with one fragment per line.  This allows you to inspect the html output.  Using scala.xml to prettyprint the output would somtimes render the html invalid.
+    @hl.highlight("""
```

Comment:
No need to use triple-quoted strings, just indent it and it'll do the right thing. The rest of the code does this